addons/calendar/views/mail_activity_views.xml

```diff
@@ -7,26 +7,26 @@
         <field name="inherit_id" ref="mail.mail_activity_view_form_popup"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='date_deadline']" position="attributes">
-                  <attribute name="attrs">{'invisible': [('activity_category', '=', 'meeting')]}</attribute>
+                  <attribute name="attrs">{'invisible': [('activity_category', 'in', ['meeting', 'scheduled_phonecall'])]}</attribute>
             </xpath>
             <xpath expr="//field[@name='user_id']" position="attributes">
-                  <attribute name="attrs">{'invisible': [('activity_category', '=', 'meeting')]}</attribute>
+                  <attribute name="attrs">{'invisible': [('activity_category', 'in', ['meeting', 'scheduled_phonecall'])]}</attribute>
             </xpath>
             <xpath expr="//button[@name='action_close_dialog']" position="attributes">
-                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('id', '!=', False)]}</attribute>
```

Comment:
I'm pretty sure you messed the conditions with the "id" and the "chaining_type" by putting "AND" instead of the "OR" that was in the code before?